espresso-support/demo/src/androidTest/java/com/novoda/movies/MovieItemViewTest.java

```diff
@@ -0,0 +1,140 @@
+package com.novoda.movies;
+
+import android.support.test.filters.LargeTest;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.novoda.espresso.HitCounter;
+import com.novoda.espresso.ViewActivityRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static org.hamcrest.CoreMatchers.is;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class MovieItemViewTest {
+
+    private static final Movie EDWARD_SCISSORHANDS = new Movie("Edward Scissorhands");
+    private static final Movie NOT_EDWARD_SCISSORHANDS = new Movie("NOT Edward Scissorhands");
+
+    @Rule
+    public ViewActivityRule<MovieItemView> viewActivityRule = new ViewActivityRule<>(R.layout.test_movie_item_view);
+
+    private HitCountingMovieItemViewListener hitCountingListener;
+
+    @Before
+    public void setUp() {
+        MovieItemView view = viewActivityRule.getView();
+
+        hitCountingListener = new HitCountingMovieItemViewListener();
+        view.attachListener(hitCountingListener);
```

Comment:
If your custom view has a separate method for attaching a listener, you can attach it in the `setUp()` method of the test, since it shouldn't be data dependent.

If it's passed in to the `bind()`/`update()` method, you can pass it at the point of binding instead, no big deal.